src/components/Navbar.js

```diff
@@ -0,0 +1,25 @@
+import link_image from '../assets/img/icon/InPlease.svg'
```

Comment:
Miss comments

// Dependencies <- dependencies as react, useState or useEffect

// Components <- just components

// Assets  <- img, icons or styles

Reply:
This need to be resolve @yormanegc